### PR TITLE
Add documentation on how to support new database in Astro Python SDK

### PR DESCRIPTION
**Pull request by pankajkoti**
closes: #682

### CODE REVIEW

File: python-sdk/docs/development/DEVELOPMENT.md [hunk 1]
```diff
@@ -202,3 +202,82 @@ You can configure the Docker-based testing environment to test your DAG
...
+   [.github/workflows/ci-python-sdk.yaml](https://github.com/astronomer/astro-sdk/blob/main/.github/workflows/ci-python-sdk.yaml).
+7. Add the database as Python SDK supported database in the [test_constants.py](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/tests/test_constants.py) module's [test_supported_database()](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/tests/test_constants.py#L23) method implementation.
+
+With the above configuration set, you can now proceed for the implementation of supporting all the SDK operators the new database.
+The purpose of each of the operators can be found in the [Astro SDK Python - Operators](https://astro-sdk-python.readthedocs.io/en/stable/operators.html) document.
+As described before, you can use test driven development and run the tests for the operators one by one located in the `tests` directory.
+By default, the [base class](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/databases/base.py)
+implementation methods will be used for the database in the tests. You will need to override some of these base class methods for the database
```
Comment (phanikumv) [inline python-sdk/docs/development/DEVELOPMENT.md:267]:
```suggestion
implementation methods will be used for the database in the tests. You will need to override some of these methods 
```

File: python-sdk/docs/development/DEVELOPMENT.md [hunk 2]
```diff
@@ -202,3 +202,82 @@ You can configure the Docker-based testing environment to test your DAG
...
+   [.github/workflows/ci-python-sdk.yaml](https://github.com/astronomer/astro-sdk/blob/main/.github/workflows/ci-python-sdk.yaml).
+7. Add the database as Python SDK supported database in the [test_constants.py](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/tests/test_constants.py) module's [test_supported_database()](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/tests/test_constants.py#L23) method implementation.
+
+With the above configurations set, you can now proceed for the implementation of supporting all SDK operators in the new database.
+The purpose of each of the operators can be found in the [Astro SDK Python - Operators](https://astro-sdk-python.readthedocs.io/en/stable/operators.html) document.
+As described before, you can use test driven development and run the tests for the operators one by one located in the `tests` directory.
+By default, the [base class](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/databases/base.py)
+implementation methods will be used for the database in the tests. You will need to override some of these methods.
```
Comment (pankajkoti) [inline python-sdk/docs/development/DEVELOPMENT.md:267]:
```suggestion
implementation methods will be used for the database in the tests. You will need to override some of these methods
```

File: python-sdk/docs/development/DEVELOPMENT.md [hunk 3]
```diff
@@ -202,3 +202,82 @@ You can configure the Docker-based testing environment to test your DAG
 
 * `dev/dags/` - DAG Files
 * `dev/logs/` - Logs files of the Airflow containers
+
+
+## Adding support for a new database
+
+You can use Test Driven approach for adding support for a new database to the Astro Python SDK. You can fulfil the
+[tests](https://github.com/astronomer/astro-sdk/tree/main/python-sdk/tests) for all the
```
Comment (feluelle) [inline python-sdk/docs/development/DEVELOPMENT.md:210]: For referencing internal files, use [relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links).

File: python-sdk/docs/development/DEVELOPMENT.md [hunk 4]
```diff
@@ -202,3 +202,82 @@ You can configure the Docker-based testing environment to test your DAG
...
+Python SDK operators by adding parameters to the existing tests for the database and additionally adding more database specific
+implementation tests. You can take a look at this [PR](https://github.com/astronomer/astro-sdk/pull/639) and
+[PR](https://github.com/astronomer/astro-sdk/pull/753) on how to add parameters for the existing tests for all the
+operators.
+
+To start with you can take the following steps for the initial configuration:
+1. Add the database name constant to the [Database](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/constants.py#L42) class in the `constants.py` module.
```
Comment (feluelle) [inline python-sdk/docs/development/DEVELOPMENT.md:217]: If we point to specific lines we should reference specific commits - not the main branch, which is constantly moving.

**Review by feluelle — COMMENTED** [review]
For referencing

File: python-sdk/docs/development/DEVELOPMENT.md (hunk 3 above)
Comment (pankajkoti) [reply python-sdk/docs/development/DEVELOPMENT.md]: done, thanks.

File: python-sdk/docs/development/DEVELOPMENT.md (hunk 4 above)
Comment (pankajkoti) [reply python-sdk/docs/development/DEVELOPMENT.md]: makes sense, done, thanks.

**Review by feluelle — APPROVED** [review]
Looks nice! :)

**Pull request merged**